src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
```

Comment:
```suggestion
  static alive = [];
  
  constructor(name) {
    this.health = 100;
    this.name = name;
    
    Animal.alive.push(this);
  }
```

Reply:
Let's split parts of code by empty line to visually separate it

// variables/properties assigning
-- empty line --
// methods creating
-- empty line
// methods calling